src/send/mod.rs

```diff
@@ -43,7 +41,7 @@ struct Email {
 }
 
 impl FromData for Email {
-    type Error = ValidationError;
+    type Error = ();
```

Comment:
I guess this will be changed once we use the suggested error crates?

Reply:
Possibly, although I think it's more contingent on a resolution to SergioBenitez/Rocket#596 tbh.